tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/RecordingType.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Azure.Sdk.Tools.TestProxy.Common
+{
+    public enum RecordingType
+    {
+        IN_MEMORY,
+        FILE_PERSISTED
```

Comment:
```suggestion
        InMemory,
        FilePersisted
```

NIT: .NET Enum values should use PascalCase.

Reply:
Other references also need to be fixed. Resolving locally, but taking feedback.